ttnn/cpp/pybind11/device.hpp

```diff
@@ -30,6 +31,37 @@ void py_module(py::module& module) {
 
     module.def("disable_and_clear_program_cache", &ttnn::disable_and_clear_program_cache, py::arg("device"), py::kw_only());
 
+    module.def("SetDefaultDevice", &ttnn::operations::experimental::auto_format::AutoFormat::SetDefaultDevice);
```

Comment:
codestyle question. Should it be "set_default_device"?

Reply:
Hi @dmakoviichuk-tt .
I followed the previous code style, but I'll confirm it to be sure. 
Thank you!

Reply:
Please update PR description with a new added functions.

Reply:
Hi @dmakoviichuk-tt,
I didn't add any new functions, just migrated the existing ones.